account-kit/core/src/types.ts

```diff
@@ -103,6 +103,15 @@ export type CreateConfigProps = RpcConnectionConfig & {
   storage?: (config?: { sessionLength: number }) => Storage;
 
   connectors?: CreateConnectorFn[];
+
+  /**
+   * If set, calls `preparePopupOauth` immediately upon initializing the signer.
+   * If you intend to use popup-based OAuth login, you must either set this
+   * option to true or manually ensure that you call
+   * `signer.preparePopupOauth()` at some point before the user interaction that
+   * triggers the OAuth authentication flow.
+   */
+  preparePopupOauthOnStartup?: boolean;
```

Comment:
nit: should we call this something like `usePopupOauth` or `usePopup` or even just `popup`, so we  don't leak impl details to devs?

Reply:
I think you're right, I'll keep it simple. If an advanced dev wants to customize this behavior, they can `useSigner` and call `preparePopupOauth` on the signer, but we don't need to go out of our way to expose this.